packages/desktop/public/index.html

```diff
@@ -8,7 +8,7 @@
             content="
                 default-src 'self';
                 connect-src 'self' https://* wss://relay.walletconnect.com;
-                img-src 'self' https://tideprotocol.infura-ipfs.io data:;
+                img-src 'self' https://tideprotocol.infura-ipfs.io https://assets.coingecko.com/coins/images data:;
```

Comment:
```suggestion
                img-src 'self' https://tideprotocol.infura-ipfs.io https://assets.coingecko.com data:;
```

With the previous CSP, the following error shows up on my side:

```
Refused to load the image 'https://assets.coingecko.com/coins/images/32429/small/lum.jpg?1698121504' because it violates the following Content Security Policy directive: "img-src 'self' https://tideprotocol.infura-ipfs.io https://assets.coingecko.com/coins/images data:".
```

This adjustment solves the problem on my end.